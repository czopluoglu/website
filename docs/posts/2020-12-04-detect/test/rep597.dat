01010100111110101111
11010001001101101110
10100000001100010110
01000100101010100010
11010001001010100110
11011111011001000011
11010001101111100000
11001111010010110111
00011001111011111111
10110011011111101000
01101101111001110001
01100001001110110110
01101001001011010110
01100100010100110011
10000100011111100111
11010101111000100110
10001001010110110100
00110101000011101011
00001000100000000100
01000000000010100001
10001001101101111111
01101100101110010011
10101001101101100111
00110011111010101110
00111110111011111001
11011000001011100011
11010011101111111101
10101101010110100000
01100000111001001010
10101001011111111111
11010110111111100001
10011011010111111110
11011111101011111110
11111011010111000000
11001100000111001110
01100100101100111110
01100001001000001110
01000101110110101110
11000001001010101011
10001001010000000010
10111101111101110111
10101101001010001011
00000011110011111111
01111111101000000100
01110111101100000100
01011111101010101110
00000100101110101110
11101111111111111111
10100110111101101111
10000000011101110010
01010000011011000100
01010001001010110110
10100001100010110010
10110001011101011001
00010000100111101111
11001010001000100100
00001011111001111100
10101110111111100111
10100001111010100100
10001001001110101010
10101010001111101111
11111001111011000011
11010001001011000000
11000110100111110111
11101101001000000010
11011111011000100010
11010010101000101110
00000001111110101001
11110111101011111011
00001001001100101010
10110001111110011101
10110111100001101110
00001000100100000010
11110001101110111111
01000001100010110001
01101110001111001111
00101000000110000010
00000000001011101111
11000000101111111111
00100000000100101110
00101110101011111100
11111110010100111111
01101101101110001101
11000000001010111110
11100111111110101111
01101100111011111111
10111001101010010110
11100011111111011110
11000100011100000110
10000100001001111011
01110101100010010110
10010101111101000111
01101111111001101110
11101101010001000100
10011111111111101110
11111111111111111110
10101101101111001101
00110011000111110001
01110000001000010110
00011000101011100001
10111111101011001010
10110011111000111110
11100001101000010111
11010001101010101101
10110111101111110010
11011101111011010000
11100001101100111111
11001001111110010111
11100011111000110110
11110011111100111011
01001000011001000011
00001110001111111111
11010001010110011000
01100011010000010111
11001010101110110111
01000100101000101111
11101111101001000100
10110001111111111000
10100000011100000100
11111110111111100111
01101010111010101111
10101010111000111110
10000101011000001100
00110001111110010110
11010010100110111010
10010011111000011111
00001111110111101011
11110101011101101101
11011001100001101100
11100100111110101011
11000000110111010101
11100101111110111011
01100001011101111111
00000111001000101001
01000101101111101111
00111111101010000011
10000101010110001101
11011101100010000110
01100101100010000000
10110111111111111111
11111101111101011111
11110111010111000111
11000000001101100111
11010101101110110010
01000001000010110011
10011101000111111111
10111111110111010101
10010001010100000111
00000000111011100110
11100011111111110101
01000111010000000011
00100000001101011100
10100101111011111110
11100011111011000010
01010101011000000000
11000010000000000111
01010000000110110000
11100101101010101101
01111101011101111111
11011111111110010111
00001001000010110110
00000001111111101110
11110011111010010111
00100100111111111110
10110001111110111101
01000001011110101011
01100100000001111111
10000010111101111010
11111111101111110101
10111101011111000101
11000001011110111100
11010000001000011110
00000000011000111010
01100100010010101111
00001100001100000110
00101101000000001110
11001001001111111111
00000010000000000010
11000000101111110011
10000101000000010010
01000000101111101110
10001001101101101100
10110000101011010110
11010000101111110111
10100001111110010111
10000001001110100111
01011011001000000111
10010011111111110110
10000100100111100011
10101000001110111100
11101011100011001101
01011011101110111011
11111001111010101110
11101011011001100100
00101100101101101110
11010000001110101101
10011111111111001111
00101111101101111111
10110011111001001000
10000000011110110011
10010100011110010010
01000101111000111010
00001000010000100110
10011111111010101110
10101000001001100110
00011110110010100011
10100101111011100110
00010100101010010010
00010001001110101110
11110111011111110011
00011011001101111101
11111111111001000111
01001001000100010000
01100011101011101111
01001001010010101111
11110101100011101100
00100100101110000001
11111111111011111001
11111001111101001100
01000110111110101011
11110101111110010111
00011000001000110010
10011000011101101111
00000001101110011001
01110111101001100110
11101110011111111110
00011101111111100011
10111001110111111111
00101000101110111110
10000011001110111111
10110011011001111110
00111101001101000010
10110110011011111111
00101100001010000010
10001011011101011101
11101111001000100010
01110101110000110111
10000101101110011011
11000111111010101110
01011100000101100011
00100100001111000111
11001101011010110011
11111001001011111110
00111101111110101010
11010011111100010010
01101110001011001111
10010011001000010010
00111101101111110111
00000001110000000010
11001100101000101101
