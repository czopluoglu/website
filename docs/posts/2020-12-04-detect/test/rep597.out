-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep597.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.07
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8746
                   IDN index value:   0.7053
                           Ratio r:   0.6269

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    1    2    2    1    1    3    1
    3    4    4    2    3    4    3    3    3    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    4    7    8   10

   -----------CLUSTER  2-------------

    3    5    6   14

   -----------CLUSTER  3-------------

    9   11   15   17   18   19

   -----------CLUSTER  4-------------

   12   13   16   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++++---+-+--+-+-+-
   +d+++---+-----+----+
   ++d+++---+++-----++-
   +++d+++++++----+----
   ++++d+++--+-------+-
   +-+++d+-+----------+
   ---+++d+++++--+-----
   ---++-+d++----+-----
   -+-+-+++d++----+---+
   +-++--+++d-++++-+--+
   --+++-+-+-d++----++-
   +-+---+--++d+++++---
   ---------+++d+++++++
   ---------+-++d+-+--+
   ++----++-+-+++d++---
   ---+----+--++-+d--++
   +--------+-++++-d+++
   --+-------+-+---+d++
   +-+-+-----+-+--+++d+
   -+---+--++--++-++++d
 
 No cross validation for this DETECT run
